baselines/marl_benchmark/metrics/basic_handler.py

```diff
@@ -41,8 +41,8 @@ def agent_info_adapter(env_obs, shaped_reward: float, raw_info: dict):
     info["collision"] = 1 if len(env_obs.events.collisions) > 0 else 0
 
     goal = env_obs.ego_vehicle_state.mission.goal
-    goal_pos = goal.position
     ego_2d_pos = env_obs.ego_vehicle_state.position[:2]
+    goal_pos = getattr(goal, "position", ego_2d_pos)
```

Comment:
Maybe adding the same comment here will be useful : `# the position of mission goal is 2-dimensional.`